hazelcast-client-new/src/test/java/com/hazelcast/client/map/ClientMapTest.java

```diff
@@ -226,6 +226,20 @@ public void testGetAllPutAll() {
     }
 
     @Test
+    public void testPutAllWithTooManyEntries() {
```

Comment:
It does not verify exactly but this change can not be tested outside. It is only a performance improvement. I think we are OK even if there is no extra test introduced.